.github/workflows/release.yml

```
@@ -123,29 +150,54 @@ jobs:
       - name: Create source archive
         run: Compress-Archive AdbWinApi-$env:PROJECT_VERSION-src AdbWinApi-$env:PROJECT_VERSION-src.zip
+
+      - name: Finalize SBOM files
+        shell: bash
+        env:
+          REPO_NAME: ${{ github.repository }}
+        run: |
+          for arch in x86_64 x86 aarch64; do
+            python3 ./finalize_sbom.py AdbWinApi-$PROJECT_VERSION.zip \
+             "https://github.com/$REPO_NAME/releases/download/$PROJECT_VERSION/AdbWinApi-$PROJECT_VERSION.zip" \
+             AdbWinApi-$PROJECT_VERSION-$arch-sbom.cyclonedx.json 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)